pkg/knode-manager/adapters/k8s/pvc.go

```diff
@@ -0,0 +1,102 @@
+package k8sadapter
+
+import (
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/runtime"
+	kubeinformers "k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/klog/v2"
+
+	"github.com/kosmos.io/kosmos/pkg/knode-manager/utils"
+)
+
+type PVCAdapter struct {
+	master kubernetes.Interface
+	client kubernetes.Interface
+
+	updatedPVC chan *corev1.PersistentVolumeClaim
+
+	stopCh <-chan struct{}
+}
+
+func NewPVCAdapter(ctx context.Context, ac *AdapterConfig) (*PVCAdapter, error) {
+	adapter := &PVCAdapter{
+		master: ac.Master,
+		client: ac.Client,
+		stopCh: ctx.Done(),
+	}
+
+	cInformer := kubeinformers.NewSharedInformerFactory(adapter.client, 0).Core().V1().PersistentVolumeClaims()
+	_, _ = cInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+		//AddFunc:    adapter.addPVC,
+		UpdateFunc: adapter.updatePVC,
+		//DeleteFunc: adapter.deletePVC,
```

Comment:
why keep this line?

Reply:
deleted, I thought it might be used later at first